src/components/tasks/card/index.tsx

```diff
@@ -258,13 +262,171 @@ const Card: FC<Props> = ({
+                            role="button"
+                            tabIndex={0}
+                        >
+                            {cardDetails.assignee}
+                        </span>
+                    </div>
+                    <div
+                        className={`${classNames.taskTagLevelWrapper} ${
+                            shouldEdit && classNames.editMode
+                        }`}
+                    >
+                        <div className={classNames.taskTagLevelContainer}>
+                            {taskTagLevel?.map((item) => (
+                                <span
+                                    key={item.tagId}
+                                    className={classNames.taskTagLevel}
+                                >
+                                    {item.tagName}{' '}
+                                    <small>
+                                        <b>LVL:{item.levelValue}</b>
+                                    </small>
+                                    {shouldEdit && isUserAuthorized && (
+                                        <span>
+                                            <button
+                                                className={
+                                                    classNames.removeTaskTagLevelBtn
+                                                }
+                                                onClick={() =>
+                                                    updateTaskTagLevel(
+                                                        item,
+                                                        'delete'
+                                                    )
+                                                }
+                                            >
+                                                &#10060;
+                                            </button>
+                                        </span>
+                                    )}
+                                </span>
+                            ))}
+                        </div>
+                        {shouldEdit && isUserAuthorized && (
+                            <TaskLevelEdit
+                                taskTagLevel={taskTagLevel}
+                                updateTaskTagLevel={updateTaskTagLevel}
+                            />
+                        )}
+                    </div>
+                </div>
+
+                {isUserAuthorized && showEditButton && (
+                    <div
+                        className={classNames.editButton}
+                        data-testid="edit-button"
+                    >
+                        <Image
+                            src="/pencil.webp"
+                            alt="edit Pencil"
+                            width={iconWidth}
+                            height={iconHeight}
+                            onClick={onEditEnabled}
```

Comment:
See if this is reachable via keyboard. If not, then should add tabIndex=0 here probably.

Reply:
yes checked it was not added tabIndex